build/ci-deploy.yml

```diff
@@ -40,7 +40,11 @@ stages:
   - provisionEnvironment
   - DockerBuild
   jobs:
-  - template: ./jobs/add-aad-test-environment.yml
+  - job: setup
+    pool:
+      vmImage: '$(WindowsVmImage)'
```

Comment:
>WindowsVmImage [](start = 18, length = 14)

Powershell not working on non-windows machines?

Reply:
It requires higher version to run in other platform. And in our case, it fails to install AzureAD module without this platform selection.

Reply:
This looks ok, I believe the Connect-AAD call in this job required Windows atm